database/seeders/EventsErstiwocheSeeder.php

```diff
@@ -367,14 +368,15 @@ public function runSport(): void
         $event->description = '<p>Auch sportliche Aktivitäten kommen bei uns nicht zu kurz. Für eine Anmeldegebühr von <strong>5€</strong> könnt ihr euch am Freitag auspowern.</p>
             <p>Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben.</p>
             <p>Bitte beachtet auch die folgenden Hinweise zu den einzelnen Programmpunkten:</p>
-            <p><strong>Fußball, Volleyball, Basketball:</strong> Die Anmeldegebühr bekommt ihr zurückerstattet, wenn es wegen schlechtem Wetter ausfällt.</p>
+            <p><strong>Fu0ball:</strong>Bitte bringt 5€ an Pfand mit</p>
             <p><strong>Yoga:</strong> Bitte bringt eine eigene Yogamatte mit.</p>
+            <p><strong>Bouldern;</strong> Falls nicht vorhanden, können Stoppersocken vor Ort für 3€ erworben werden.</p>
             <p><strong>Allgemein:</strong> Anschließende Teilnahme an den Kulturprogrammpunkten ist nur die “Kebabtour” zeitlich möglich.</p>
```

Comment:
```suggestion
            <p><strong>Allgemein:</strong> Eine anschließende Teilnahme an weiteren Programmpunkten ist nur mit der “Foodtour” zeitlich möglich.</p>
```

Reply:
This must be validated, there are new time slots for each event. Please see the "schichtplan".

Reply:
time slots:

| slot                        | time               |
|---------------------|----------------|
| Fußball, Volleyball | 12:00 - 15:00 |
| Bouldern               | 13:00 - 16:00 |
| Yoga                      | 12:00 - 13:30 |
| Lasertag                | 12:20 - 14:40 |
| Foodtour               | 15:30 - 17:00 |
| Tierpark                 | 15:30 - 17:00 |